Apps/SystemTools/Menu/INPUTD (value)/Patched_14ms.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
+cd "$(dirname "$0")"
+
+source .switcher.sh
```

Comment:
This line in all three scripts was not working on my device. I had to modify it like so: 

```suggestion
source "$(pwd)/.switcher.sh"
```

Something in my environment was causing confusion with the relative path. Adding the full working directory fixed it on my device.

Reply:
I honestly don't see any reason for this to fix an issue but if it really fix it for you i guess i can use the absolute path. Thanks.